.github/workflows/go.yml

```diff
@@ -46,6 +46,9 @@ jobs:
         # Use blst tag to allow go and bazel builds for blst.
         run: go build -v --tags=blst_enabled ./...
 
+      - name: Test static analyzers
+        run: go test -v ./tools/analyzers/...
```

Comment:
Also, do static analyzers need to be w/i `Build`? I mean, if static analyzers find some problems it is not build failed, it is some test failed. Can we probably move it into `Check` (or its own `Test` section)? Are there any cons?

Reply:
I agree that it would be better to move these tests into a separate `Test` section.